### PR TITLE
Add tests for subread #106

### LINKED ISSUE #106: Add tests for subread

### PR DESCRIPTION
**Pull request by ntoda03**
## PR checklist

Closes #106  

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] Remove all TODO statements.
- [x] Follow the naming conventions.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: software/subread/featurecounts/meta.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      type: file
+      description: Summary log file
+      pattern: "*.featureCounts.txt.summary"
+  - version:
+      type: file
+      description: File containing software version
+      pattern: "*.{version.txt}"
+
+authors:
+  - "@ntoda03"
```
Comment (Emiller88) [inline software/subread/featurecounts/meta.yml:52]:
```suggestion
  - "@ntoda03"
  - "@drpatelh"
```

**Review by Emiller88 — CHANGES_REQUESTED** [review]
Awesome! I was just thinking that these were needed last night. Just need to add @drpatelh as an author I think, and probably rebase the commits or clean them up.

File: software/subread/featurecounts/meta.yml (hunk 1 above)
Comment (ntoda03) [reply software/subread/featurecounts/meta.yml:52]: Right! He was in there for a different PR. I've cleaned it up to get rid of that.

**Review by drpatelh — APPROVED** [review]
👌🏽

**Pull request merged**